.github/scripts/common.py

```diff
@@ -4,16 +4,23 @@
 from fabric.api import *
 import requests
 from local_flags import RUN_LOCAL, local_fsim_dir
+from ci_variables import ci_firesim_dir
 
 # Reuse manager utilities
 sys.path.append(local_fsim_dir + "/deploy/awstools")
 from awstools import get_instances_with_filter
 
+# Github URL related constants
+gh_repo_path        = "firesim/firesim"
+gha_api_url         = f"https://api.github.com/repos/{gh_repo_path}/actions"
```

Comment:
This can be constructed from `$GITHUB_API_URL`. See https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
See above run_local comment.